tests/failpoints/cases/test_table_properties.rs

```diff
@@ -167,6 +168,7 @@ fn do_gc(
 }
 
 #[test]
+#[cfg(not(feature = "test-engines-panic"))]
```

Comment:
Why not `test-engine-kv-rocksdb`. The problem with `test-engines-panic` is, the build will still fail if we uses `--features test-engine-kv-panic,test-engine-raft-rocksdb` or `--features test-engines-another-kind-of-engine`.

Reply:
It seems I can't express that way.  When ENABLE_FEATURES is not set by default, config predicate feature=test-engine-kv-rocksdb won't be satisfied either.